types/google.visualization/index.d.ts

```diff
@@ -654,6 +672,11 @@ declare namespace google {
             reverseCategories?: boolean;
             selectionMode?: string // single / multiple
             series?: any;
+            domainAxis?: { type: string };
+            trendlines?: { [key: number]: any; };
```

Comment:
Please create an interface for all propertied that `trendlines` can have.

Reply:
updated